WSL/connect-usb.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Connect USB devices to WSL 2 
+description: Learn how to connect a USB device to your WSL 2 Linux distribution using usbipd-win.
+ms.date: 11/09/2021
+ms.topic: article
+ms.localizationpriority: medium
+---
+
+# Connect USB devices to WSL 2
+
+This guide will walk through the steps necessary to connect a USB device to a Linux distribution running on WSL 2 using the USB/IP open-source project, [usbipd-win](https://github.com/dorssel/usbipd-win).
```

Comment:
```suggestion
This guide will walk through the steps necessary to connect a USB device to a Linux distribution running on WSL using the USB/IP open-source project, [usbipd-win](https://github.com/dorssel/usbipd-win).
```

Reply:
Based on our discussion yesterday, deleting this change as USB support is only for WSL 2 (and won't work for WSL 1 from what I understand).